src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/UninstallPackageCommand.cs

```diff
@@ -54,12 +55,30 @@ private void Preprocess()
 
         protected override void ProcessRecordCore()
         {
+            var startTime = DateTime.Now;
+            ActionStopWatch.Restart();
+
             Preprocess();
 
             SubscribeToProgressEvents();
             Task.Run(() => UnInstallPackage());
             WaitAndLogPackageActions();
             UnsubscribeFromProgressEvents();
+
+            ActionStopWatch.Stop();
+            var actionTelemetryEvent = TelemetryUtility.GetActionTelemetryEvent(
+                new[] { Project },
+                NugetOperationType.Uninstall,
+                OperationSource.PMC,
+                startTime,
+                _status,
+                _errorMessage,
+                1,
```

Comment:
Literal arguments reduce readability:  what does 1 signify?

Options to improve readability include:

1.  Promote the argument to a well-named local:  var packageCount = 1;
2.  Inline comment with the parameter name:  `/*packageCount*/ 1`
3.  Named parameter:  `packageCount: 1`, but unfortunately subsequent arguments would also require named parameters.